pom.xml

```diff
@@ -239,14 +239,9 @@
 
     <distributionManagement>
         <snapshotRepository>
-            <id>sonatype-nexus-snapshots</id>
+            <id>ossrh</id>
```

Comment:
The Maven settings and staging plugin use `ossrh` as the id. My guess is that you copied these configurations from two different sources.